source/change-streams/tests/README.rst

```diff
@@ -155,12 +155,12 @@ The following tests have not yet been automated, but MUST still be tested. All t
 #. ``ChangeStream`` will throw an exception if the server response is missing the resume token (if wire version is < 8, this is a driver-side error; for 8+, this is a server-side error)
 #. After receiving a ``resumeToken``, ``ChangeStream`` will automatically resume one time on a resumable error with the initial pipeline and options, except for the addition/update of a ``resumeToken``.
 #. ``ChangeStream`` will not attempt to resume on any error encountered while executing an ``aggregate`` command. Note that retryable reads may retry ``aggregate`` commands. Drivers should be careful to distinguish retries from resume attempts. Alternatively, drivers may specify `retryReads=false` or avoid using a [retryable error](../../retryable-reads/retryable-reads.rst#retryable-error) for this test.
-#. ``ChangeStream`` will not attempt to resume after encountering error code 11601 (Interrupted), 136 (CappedPositionLost), or 237 (CursorKilled) while executing a ``getMore`` command.
```

Comment:
Is this being removed because it is redundant with the new "change stream errors on MaxTimeMSExpired" test? This test would still pass since there won't be a resumable label on any of these errors, right?

Reply:
This test would still pass. I removed it because it's explicitly calling out the codes that were on the old blacklist. In my opinion, this doesn't add any coverage outside of the new spec test that you mentioned. Also, it makes these three errors seem special even though they're not, which could confuse someone reading the spec for the first time in the future if they don't have the all of the past context.

Reply:
That seems reasonable, just double checking.